tests/e2e/testPages/prebid-body-multi-players.html

```diff
@@ -110,7 +110,7 @@
 	    
 	    function pageNotification2(type, data) {
 			if (type === 'event') {
-	    		appendStatistics('Event: ' + data);
+	    		appendStatistics('Event: ' + data, 2);
```

Comment:
likewise - we need a comment stating the purpose of the numerical digit

Reply:
added comment